docs/format.md

```diff
@@ -101,7 +101,7 @@ Some languages have dialects:
 <pre>
 load("@aspect_rules_lint//format:defs.bzl", "format_multirun")
 
-format_multirun(<a href="#format_multirun-name">name</a>, <a href="#format_multirun-jobs">jobs</a>, <a href="#format_multirun-print_command">print_command</a>, <a href="#format_multirun-disable_git_attribute_checks">disable_git_attribute_checks</a>, <a href="#format_multirun-kwargs">kwargs</a>)
+format_multirun(<a href="#format_multirun-name">name</a>, <a href="#format_multirun-jobs">jobs</a>, <a href="#format_multirun-print_command">print_command</a>, <a href="#format_multirun-disable_git_attribute_checks">disable_git_attribute_checks</a>, <a href="#format_multirun-kwargs">**kwargs</a>)
```

Comment:
What caused this?

Reply:
It's a change in bazel 8, see https://github.com/bazel-contrib/bazel_features/pull/75